### DISCUSSION TITLE
Rendering subdirectoy  .md files onto parent page

### DISCUSSION THREAD

**Discussion by thinkbig1979** [nuxt/content#1169] [Q&A]
Hi all, I'm using the wind starter as a base for building a website that explains my services and shows testimonials.
Getting the top level navigation working was easy enough, but in my "/content/3.testimonials" folder I have a bunch of MD files, one for each testimonial. I'd like to display the testimonials as cards on the /testimonials page, and figured I could use the ContentList component to grab the content from each testimonial and render it as I please. However, I've tried building a really simple Vue component, called "MyTestimonials.vue" , like this:

```vue
<template>
  <main>
    <ContentList v-slot="{ list }" path="/3.testimonials">
      <div v-for="article in list" :key="article._path">
        <h2>{{ article.title }}</h2>        <p>{{ article.description }}</p>
      </div>
    </ContentList>
  </main>
</template>
``` 

I dropped it into /components/content/
 
Adding

```md
:my-testimonials   
``` 

to the index.md file in /content/3.testimonials produces nothing on the Testimonials page.

Adding this in stead 

```md
::my-testimonials:: 
``` 
produces `({ data }) => emptyNode("not-found", data)` on the Testimonials page 

Not sure if my approach is altogether flawed, or if it's something small.
Any input would be appreciated.
Thanks in advance.

**Comment by avxkim** [comment]
Faced the same issue, this module looks bugged.

**Comment by farnabaz** [comment]
This issue is resolve in main branch and waiting for release, until then you can use `@nuxt/content-edge` 

```diff
- "@nuxt/content": "^2.0.0",
+ "@nuxt/content": "npm:@nuxt/content-edge@latest",
```

**Reply by thinkbig1979** [reply]
Thanks @farnabaz

**Reply by thinkbig1979** [reply]
Hmm, just tried with release 2.0.1. (Deleted node modules and package-lock.json, then updated 
 ` "@nuxt/content": "^2.0.0"`  to
` "@nuxt/content": "^2.0.1"`  and ran npm install.

Here's my file structure:
![image](https://user-images.githubusercontent.com/18327273/171870270-3684bf28-ebac-4d8e-b371-dbf9510901e1.png)

Now, regardless of whether I use the single or double colon invocation of the component, all I'm seeing is the title text and body text of the first markdown file it encounters in the `3.testimonials` folder outside of index.md. 
The component is being completely ignored it seems.
